packages/swingset-runner/src/chain.js

```diff
@@ -4,7 +4,7 @@ import { makeFakeStorageKit } from '@agoric/internal/src/storage-test-utils.js';
 import * as STORAGE_PATH from '@agoric/internal/src/chain-storage-paths.js';
 import { extractCoreProposalBundles } from '@agoric/deploy-script-support/src/extract-proposal.js';
 
-const { Fail, quote: q } = assert;
+import { Fail, q } from '@endo/errors';
```

Comment:
Should remove space between this and the imports block, and maybe float above `@agoric` imports depending on how much we like layer-caking our imports.

Reply:
Yes, I like have imports be vaguely in layering order. So I made a layered import order pass over most occurrences on @endo/errors.

Hugely done!